app/assets/stylesheets/application.css.scss

```diff
@@ -86,7 +86,12 @@ article {
 }
 
 a, a:visited, a:link {
-    color: white;
+  color: navy;
```

Comment:
Color `navy` should be written in hexadecimal form as `#000080`<br>Color literals like `navy` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own line